pkgs/development/python-modules/syndom/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib
+, fetchFromGitLab
+, buildPythonPackage
+, pkg-config
+, meson
+, ninja
+, isPy3k
+, pugixml
+, fmt
+, html-tidy
+, pybind11
+}:
+
+buildPythonPackage rec {
+  pname = "syndom";
+  version = "unstable-2023-03-25";
+
+  format = "other";
+
+  src = fetchFromGitLab {
+    owner = "gabmus";
+    repo = "syndication-domination";
+    rev = "75920321062d682437f3fb0319dad227d8b18f6c";
+    sha256 = "sha256-fOlE9CsNcmGkVBXaqYHxLDWB8voeRp46+dZYIJIwg7o=";
+  };
+
+  # Python 2.x is not supported.
+  disabled = !isPy3k;
```

Comment:
We should use pythonOlder straight away because 3.4 probably also does not work. Also normally we put disabled next to format

Reply:
You're right pybind11 says it supports Python 3.6+. I based this off of gst-python and other modules with `format = "other"` which have the attribute set after `src`. Moved it to be below `version`.

[pkgs/development/python-modules/syndom/default.nix](https://github.com/NixOS/nixpkgs/blob/aa087eb3af9621b2a062853d8cb1e36b8e4e611a/pkgs/development/python-modules/gst-python/default.nix)